packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -72,6 +81,12 @@ mkdir -p %{buildroot}%{_bindir}
 cp -pa .%{_bindir}/foreman-node %{buildroot}%{_bindir}/foreman-node
 cp -pa .%{gem_instdir}/sbin/upload-salt-reports %{buildroot}%{_sbindir}/upload-salt-reports
 install -Dp -m0644 .%{gem_instdir}/cron/smart_proxy_salt %{buildroot}%{_sysconfdir}/cron.d/%{gem_name}
+mkdir -p %{buildroot}%{smart_proxy_dynflow_bundlerd_dir}
```

Comment:
I think we do the same thing in ansible, but I'm wondering if this actually belongs in the `rubygem-smart_proxy_salt_core` package. That's the actual plugin to dynflow rather than this package.

Reply:
We do the same thing in ansible, I just wanted things to be consistent. If being inconsistent is not an issue, I wouldn't mind changing it.

Reply:
@evgeni thoughts? perhaps also something we should fix in ansible?